net/faq/_index.md

```diff
@@ -5,10 +5,518 @@ type: docs
+How to make a valid PDF/A document unless the missing font or its substitution is provided?
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+To create a valid PDF/A document in Aspose.PDF for .NET, you need to ensure that all required fonts are embedded or substituted. Here's an example:
+
+- Use `Convert` to ensure the document meets PDF/A standards.
+
+```csharp
+// Load the document
+using (Document pdfDoc = new Document("input.pdf"))
+{
+    PdfFormatConversionOptions options = new PdfFormatConversionOptions("conversion-log.xml", PdfFormat.PDF_A_1B, ConvertErrorAction.Delete);
+
+    // Substitute all missing fonts with the default one
+    options.FontEmbeddingOptions.UseDefaultSubstitution = true;
+
+    // Convert to PDF/A
+    pdfDoc.Convert(options);
+
+    // Save as PDF/A
+    pdfDoc.Save("output-pdfa.pdf");
+}
+```
+        </p>
+    </div>
+
+    <div itemscope itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question">
+        <h3 itemprop="name" class="faq-q">
+I see errors in PDF/A conversion log. Does it mean that the document wasn't converted successfully?
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+No, Aspose.PDF logs all problems it encountered, including the ones that were automatically fixed. If all entries in the log are marked as Convertable="True", it means that all problems were fixed, and the document was successfully converted. Only the entries with Convertable=False indicate the conversion failure.
+        </p>
+    </div>
+
+    <div itemscope itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question">
+        <h3 itemprop="name" class="faq-q">
+Does Aspose.PDF for .NET support Linux?
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+Yes, Aspose.PDF for .NET supports running on Linux environments. You can use the .NET Core version or later, which is cross-platform and can be used on Windows, macOS, and Linux.
+        </p>
+    </div>
+
+    <div itemscope itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question">
+        <h3 itemprop="name" class="faq-q">
+Does Aspose.PDF for .NET support .NET 2.0, 3.5, and 4.0 frameworks?
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+Aspose.PDF for .NET does not support the following .NET frameworks:
+
+- .NET 2.0
+- .NET 3.5
+
+But you can use Aspose.PDF for .NET with any of these frameworks in your project. The library provides a consistent API across the different .NET versions, making it easy to migrate your code between frameworks as needed.
```

Comment:
However, you can use Aspose.PDF for .NET with any of these frameworks for your project.